src/view/observer/focusobserver.js

```diff
@@ -8,6 +8,10 @@
  */
 
 import DomEventObserver from './domeventobserver';
+import global from '@ckeditor/ckeditor5-utils/src/dom/global';
+
+const setTimeout = global.window.setTimeout;
+const clearTimeout = global.window.clearTimeout;
```

Comment:
Can't you simple use Sinon fake timeout? http://sinonjs.org/releases/v2.0.0/fake-timers/